docs/core/docker/visual-studio-tools-for-docker.md

```diff
@@ -1,32 +1,31 @@
 ---
 title: Visual Studio Tools for Docker
 description: Using Visual Studio Tools for Docker 
-keywords: .NET, .NET Core, Docker, ASP.NET Core, Visual Studio 2015
+keywords: .NET, .NET Core, Docker, ASP.NET Core, Visual Studio
 author: spboyer
 ms.author: shboyer
-ms.date: 09/16/2016
+ms.date: 04/27/2017
 ms.topic: article
 ms.prod: .net-core
 ms.technology: dotnet-docker
 ms.devlang: dotnet
 ms.assetid: 1f3b9a68-4dea-4b60-8cb3-f46164eedbbf
 ---
 
-# Visual Studio Tools for Docker 
-Developing and debugging your application in a Docker container can be a ceremony of tasks to get setup with various tools. [Visual Studio Tools for Docker](https://visualstudiogallery.msdn.microsoft.com/0f5b2caa-ea00-41c8-b8a2-058c7da0b3e4) helps you get past the hurdles and find the bugs using F5 to debug your application directly in a locally hosted Docker Container. 
+# Visual Studio Tools for Docker
 
-> [!NOTE]
->The current version targets Linux Docker containers, with Windows Containers coming soon.
+[Microsoft Visual Studio 2017](https://www.visualstudio.com/) supports building and debugging .NET Framework web/console applications using Windows containers or .NET Core web/console applications using Linux containers. The [Docker for Windows](https://docs.docker.com/docker-for-windows/install/) install package includes everything you need to run Docker on a Windows system.
```

Comment:
We now support Windows containers, in addition to Linux containers.

The docker tools are installed as part of the .NET Core workload with the Visual Studio 2017 installer. The link isn't needed.